### PR TITLE
Decouple from Templating Component, Stage 1 

### PR DESCRIPTION
**Pull request by covex-nn**
I am targeting this branch, because this is a patch.

## Changelog

```markdown
### Changed
- Switch all templates references to Twig namespaced syntax
- Switch from templating service to sonata.templating
```

## To do

- [X] Switch all templates references to Twig namespaced syntax. Update docs, src and tests
- [x] Add `sonata-project/block-bundle` to composer.json
- [x] Switch from `templating` service to `sonata.templating`

## Subject

This PR is a part of a big plan of decoupling sonata-project bundles from Templating Component. See sonata-project/dev-kit#380 for details

### CODE REVIEW

**Comment by covex-nn** [comment]
Please, restart failed travis job

File: .travis/install_test.sh [hunk 1]
```diff
@@ -17,4 +17,4 @@ if [ "${COMPOSER_FLAGS}" = '--prefer-lowest' ]; then
     composer update --prefer-dist --no-interaction --prefer-stable --quiet
 fi
 
-composer update --prefer-dist --no-interaction --prefer-stable ${COMPOSER_FLAGS}
+travis_wait 20 composer update --prefer-dist --no-interaction --prefer-stable ${COMPOSER_FLAGS}
```
Comment (greg0ire) [inline .travis/install_test.sh]: dev-kit will make this disappear

Comment (covex-nn) [reply .travis/install_test.sh]: i just want to know how much time we need, i will remove it

Comment (covex-nn) [reply .travis/install_test.sh]: no luck =(
i do not want to debug `Composer\DependencyResolver\Solver::runSat` =)

**Pull request merged**